src/containers/Layout/Sidebar.js

```diff
@@ -16,6 +16,8 @@ import { getBigNumber } from 'utilities/common';
 import toast from 'components/Basic/Toast';
 import XVSIcon from 'assets/img/venus.svg';
 import XVSActiveIcon from 'assets/img/venus_active.svg';
+import CareerIcon from 'assets/img/career.png';
```

Comment:
Shouldn't we have a separate icon for the non-selected "Career" menu item? Now it weirdly stands out among the others — it's a bit larger and colored. Or it is intentional? 🤔

<img width="228" alt="Screenshot 2021-11-10 at 11 07 41" src="https://user-images.githubusercontent.com/10987782/141076260-da0df212-728f-4b03-9dc1-3c3c96bfdefe.png">